src/main/java/edu/harvard/iq/dataverse/export/ddi/DdiExportUtil.java

```diff
@@ -195,7 +197,15 @@ private static void createStdyDscr(XMLStreamWriter xmlw, DatasetDTO datasetDto)
         writeProducersElement(xmlw, version);
         
         xmlw.writeStartElement("distStmt");
-        if (datasetDto.getPublisher() != null && !datasetDto.getPublisher().equals("")) {
+      //The default is to add Dataverse Repository as a distributor. The excludeinstallationifset setting turns that off if there is a distributor defined in the metadata
+        boolean distributorSet=false;
+        MetadataBlockDTO citationDTO= version.getMetadataBlocks().get("citation");
+        if(citationDTO!=null) {
+            if(citationDTO.getField(DatasetFieldConstant.distributorName)!=null) {
+                distributorSet=true;
+            }
+        }
+        if (!StringUtils.isEmpty(datasetDto.getPublisher()) && !(ConfigProvider.getConfig().getOptionalValue("dataverse.export.distributor.excludeinstallationifset", Boolean.class).orElse(false) && distributorSet)) {
```

Comment:
This is DDI specific, so IMHO we should use `dataverse.export.ddi.distributor.excludeinstallationifset`

Reply:
It could/should apply to any export that does this - only the DDI, DDI HTML Cookbook and OpenAire are affected today, but I assume we'd want other formats follow this setting if they include such a field. (ORE probably should do this, doesn't yet).